src/custom/CortiAuth.ts

```diff
@@ -33,6 +35,13 @@ interface AuthorizationCodeServer {
     code: string;
 }
 
+interface AuthorizationCodePkceServer {
```

Comment:
let's maybe use `AuthorizationPkceServer` instead? Nikolas in some discussion made a valid comment, that when we usually talk about "code" it means "authorization_code", and PKCE usually just PKCE. We left it in the console because we have it the same way in the docs, but if we can at least in typings make it simpler -- I don't see why not